web/models/user.ex

```diff
@@ -19,7 +19,7 @@ defmodule Pairmotron.User do
   @minimum_password_length 8
 
   @required_params ~w(name email)
-  @optional_params ~w(password password_confirmation active role_id)
+  @optional_params ~w(password password_confirmation active is_admin)
```

Comment:
So this made me just think about something fun. I think that any user can create an admin user by manually hitting the create route... Not new to this PR, this just made me think about it. Perhaps we should add a protection around this in the user :create action validating that the logged in user is an admin?